ci-operator/step-registry/acm/fetch-managed-clusters/README.md

```diff
@@ -0,0 +1,31 @@
+# acm-fetch-managed-clusters-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+- [Requirements](#requirements)
+  - [Infrastructure](#infrastructure)
+  - [Environment Variables](#environment-variables)
+
+## Purpose
+
+To gather information about the managed clusters that are created as a result of the clc-test-create-ref.
```

Comment:
Elaborate, this step will just access the hub cluster and discover info about the managed cluster.

For the purpose of creating a managedCluster.json file.

Other tests can parse this json to make use of the managed cluster values.